templates/openstackconfiggenerator/config/17.1

```diff
@@ -0,0 +1 @@
+17.0
```

Comment:
for now we just link 17.1 -> 17.0 until we see there is a need to have dedicated 17.1 deplate.